code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1415,7 +1415,7 @@ datum
 
 			on_mob_life(var/mob/M, var/mult = 1)
 				if (!M) M = holder.my_atom
-				M.changeStatus("radiation", 2 SECONDS * mult)
+				M.take_radiation_dose(0.1 SIEVERTS * mult, internal=TRUE)
```

Comment:
```suggestion
				M.take_radiation_dose(0.02 SIEVERTS * mult, internal=TRUE)
```
should match mutagen in rad intensity. Could maybe go to `0.03`, but old slurry was less than mutagen in the first place so 🤷